randovania/games/prime1/gui/ui_files/preset_prime_qol.ui

```diff
@@ -268,6 +268,23 @@
              </property>
             </widget>
            </item>
+           <item>
```

Comment:
after looking at this in the screnshot, can you move this to be  backwards labs and backwards upper mines?

Reply:
![RemoveBarsGuiChange](https://github.com/user-attachments/assets/78bcb300-3f8f-46ac-b651-f2290204e510)